src/DeviceInfo.js

```diff
@@ -61,7 +61,9 @@ define(function(require) {
          * The device has touch events
          * @type {boolean}
          */
-        this.hasTouch = ('ontouchstart' in window);
+        // UserAgents with 'Touch' is IE on Microsoft Surfaces
+        this.hasTouch = ('ontouchstart' in window) ||
+            window.navigator.userAgent.toLowerCase().indexOf('touch') !== -1;
```

Comment:
Wouldn't hurt to link to https://msdn.microsoft.com/en-us/library/ie/hh920767%28v=vs.85%29.aspx#touch maybe...